mmv1/templates/terraform/examples/apphub_service_basic.tf.erb

```diff
@@ -0,0 +1,105 @@
+resource "google_apphub_application" "application" {
+  location = "us-east1"
+  application_id = "<%= ctx[:vars]['application_id'] %>"
+  scope {
+    type = "REGIONAL"
+  }
+}
+
+resource "google_project" "service_project" {
+  project_id ="<%= ctx[:vars]['service_project_attachment_id'] %>"
+  name = "Service Project"
+  org_id = "<%= ctx[:test_env_vars]['org_id'] %>"
+  billing_account = "<%= ctx[:test_env_vars]['billing_account'] %>"
+}
+
+# Enable Compute API
+resource "google_project_service" "compute_service_project" {
+  project = google_project.service_project.project_id
+  service = "compute.googleapis.com"
+}
+
+resource "time_sleep" "wait_120s" {
+  depends_on = [google_project_service.compute_service_project]
+
+  create_duration = "120s"
+}
+
+resource "google_apphub_service_project_attachment" "service_project_attachment" {
+  service_project_attachment_id = google_project.service_project.project_id
+  depends_on = [time_sleep.wait_120s]
+}
+
+# discovered service block
+data "google_apphub_discovered_service" "catalog-service" {
+  provider = google
+  location = "us-east1"
+  service_uri = "//compute.googleapis.com/${google_compute_forwarding_rule.forwarding_rule.id}"
```

Comment:
It looks like this lookup is causing the test to fail. Do we maybe need to increase the wait time on the sleep to ensure the discovered service exists?

Reply:
Yes, we need to wait for a few minutes after the forwarding rule is created; I've made a change wait for 120 seconds to ensure the discovered service exists.